playbook/app/pb_kits/playbook/pb_bar_graph/docs/_bar_graph_custom.md

```diff
@@ -1,2 +0,0 @@
-The `customOptions` prop provides comprehensive access to additional [Highcharts options](https://api.highcharts.com/highcharts/) that are not explicitly defined as props.
- It's important to note that certain options may require specific script imports to function properly.
```

Comment:
Hey @anthonymig88 , not sure if deleting this doc copy was intentional here?

Reply:
@nidaqg I was a bit unsure why it was included in this overall pull request, so I removed it. If it's meant to stay and I'm misunderstanding, please let me know and I'll fix it right away!

Reply:
It was probably added by mistake, I would put that doc copy back so you only have those 2 changed files!